crates/jstzd/src/task/jstzd.rs

```diff
@@ -97,6 +102,10 @@ impl JstzdConfig {
         &self.baker_config
     }
 
+    pub fn octez_rollup_config(&self) -> &OctezRollupConfig {
```

Comment:
Is this in use now?

Reply:
yes in the test! probably need it later when returing the config to the endpoint too